portal-kernel/src/com/liferay/portal/kernel/security/auth/CompanyThreadLocal.java

```diff
@@ -73,8 +86,8 @@ public static SafeCloseable lock(long companyId) {
 					currentCompanyId.longValue(), " are different"));
 		}
 
-		SafeCloseable safeCloseable = _companyId.setWithSafeCloseable(
-			companyId);
+		SafeCloseable safeCloseable = _setWithSafeCloseable(
```

Comment:
@yuhai This will cause same issue as https://issues.liferay.com/browse/LPS-182190

Reply:
@tinatian, @dantewang  I just add the commit(https://github.com/liferay-core-infra/liferay-portal/pull/4763/commits/b06930f0cbb33aa5e88535e3cab8034eaa7112b5) to resolve the issue